### PR TITLE
2.8-hf-azure-diable-prop

### PR DESCRIPTION
**Pull request by resmaa**
**2.8 Hot Fix**

Disable properties request based on the cloud account flag

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/877#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/04-clusters/1-new-clusters/03-azure.md [hunk 1]
```diff
@@ -84,15 +88,22 @@ The following steps need to be performed to provision a new Azure cluster:
     | **Region** | Select a region in Azure in which the cluster should be deployed.|
     | **Resource Group** | Select the resource group in which the cluster should be deployed.|
     | **SSH Key** | Public key to configure remote SSH access to the nodes.|
-    | **Static Placement** | By default, Palette uses dynamic placement, wherein a new VPC with a public and private subnet is created to place cluster resources for every cluster.<br /> These resources are fully managed by Palette and deleted when the corresponding cluster is deleted. Turn on the **Static Placement** option if it's desired to place resources into preexisting VPCs and subnets. If the user is making the selection of **Static Placement** of resources, the following placement information needs to be provided:
-    || **Virtual Network**: Select the virtual network from dropdown menu.
-    || **Control plane Subnet**: Select the control plane network from the dropdown menu.
-    || **Worker Network**: Select the worker network from the dropdown.
```
Comment (codrin-iftimie) [inline content/docs/04-clusters/1-new-clusters/03-azure.md:90]: In the normal flow you'd still have to select those.
If you select an cloud account with Disable properties enabled then you have to input the right details as described: 
```
Name <br /> CIDR Block <br /> Security Group Name
```

File: content/docs/04-clusters/1-new-clusters/03-azure.md [hunk 2]
```diff
@@ -84,15 +88,22 @@ The following steps need to be performed to provision a new Azure cluster:
...
+    ||**Control plane Subnet**: For Azure resource management <br /> Name <br /> CIDR Block <br /> Security Group Name
+    ||**Worker Network**: <br /> Name <br /> CIDR Block <br /> Security Group Name
+    |**Update worker pools in parallel**| Check the box to concurrently update the worker pools.|
+
+<InfoBox>
+
+If the Palette [cloud account](/clusters/new-clusters/azure#creatinganazurecloudaccount) is created with **Disable Properties** and 
+**Static Placement** checked the network informations from user's Azure account will not be imported to palette account. Hence all the network informations given in the above table needs to be filled manually by the user (no drop down menu will be available).
+
+</InfoBox>
```
Comment (codrin-iftimie) [inline content/docs/04-clusters/1-new-clusters/03-azure.md:104]: I'd enhance this note with the fact that you'd have to manually input the information for **Control plane Subnet** and **Worker Network**
```
Name <br /> CIDR Block <br /> Security Group Name
```
if these conditions are met

**Review by codrin-iftimie — CHANGES_REQUESTED** [review]

File: content/docs/04-clusters/1-new-clusters/03-azure.md (hunk 2 above)
Comment (resmaa) [reply content/docs/04-clusters/1-new-clusters/03-azure.md:104]: done

File: content/docs/04-clusters/1-new-clusters/03-azure.md [hunk 3]
```diff
@@ -84,15 +89,21 @@ The following steps need to be performed to provision a new Azure cluster:
...
     || **Virtual Network**: Select the virtual network from dropdown menu.
     || **Control plane Subnet**: Select the control plane network from the dropdown menu.
     || **Worker Network**: Select the worker network from the dropdown.
+    |**Update worker pools in parallel**| Check the box to concurrently update the worker pools.|
+
+<InfoBox>
 
-5. Make the choice of updating the worker pool in parallel.
+If the Palette [cloud account](/clusters/new-clusters/azure#creatinganazurecloudaccount) is created with **Disable Properties** and with
+**Static Placement** the network informations from user's Azure account will not be imported to palette account. Hence user can manually input the information for the ** <Tooltip trigger={<u>Control Plane Subnet</u>}><a href="">Control Plane Subnet</a> <br /> Name <br /> CIDR Block <br /> Security Group Name</Tooltip>** and the ** <Tooltip trigger={<u>Worker Network</u>}><a href="">Worker Network</a> <br /> Name <br /> CIDR Block <br /> Security Group Name</Tooltip>** (no drop down menu will be available).
```
Comment (codrin-iftimie) [inline content/docs/04-clusters/1-new-clusters/03-azure.md:102]: `<a href="">Control Plane Subnet</a>`

no need for this anchor

File: content/docs/04-clusters/1-new-clusters/04-aks.md [hunk 4]
```diff
@@ -130,8 +133,19 @@ The following steps need to be performed to provision a new cluster:
...
+    ||**Virtual Resource Group**: The logical container for grouping related Azure resources.
+    || **Virtual Network**: Select the virtual network from dropdown menu.
+    || **Control plane Subnet**: Select the control plane network from the dropdown menu.
+    || **Worker Network**: Select the worker network from the dropdown.
+    |**Update worker pools in parallel**| Check the box to concurrently update the worker pools.|
+
+<InfoBox>
+
+If the Palette [cloud account](/clusters/new-clusters/azure#creatinganazurecloudaccount) is created with **Disable Properties** and with
+**Static Placement** the network informations from user's Azure account will not be imported to palette account. Hence user can manually input the information for the ** <Tooltip trigger={<u>Control Plane Subnet</u>}><a href="/glossary-all/#users">Control Plane Subnet</a> <br /> Name <br /> CIDR Block <br /> Security Group Name</Tooltip>** and the ** <Tooltip trigger={<u>Worker Network</u>}><a href="/glossary-all/#users">Worker Network</a> <br /> Name <br /> CIDR Block <br /> Security Group Name</Tooltip>** (no drop down menu will be available).
```
Comment (codrin-iftimie) [inline content/docs/04-clusters/1-new-clusters/04-aks.md:146]: `<a href="/glossary-all/#users">Worker Network</a>`

no need for this anchor

**Review by codrin-iftimie — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/877#pullrequestreview-1107003788" title="Approved">codrin-iftimie</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/877#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**